src/net_processing.cpp

```diff
@@ -2535,6 +2541,17 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
         return;
     }
 
+    if (msg_type == NetMsgType::SENDADDRV2) {
+        if (pfrom.fSuccessfullyConnected) {
+            // Disconnect peers that send SENDADDRV2 message after VERACK; this
+            // must be negotiated between VERSION and VERACK.
```

Comment:
maybe refer to BIP155 here if you retouch; if you do I'll update #20592 similarly to refer to BIP339

Reply:
I'll do so if the BIP change is merged before this PR is.

Reply:
proposed in #20646